src/Tempest/Support/src/PathHelper.php

```diff
@@ -36,4 +43,79 @@ public static function make(string ...$paths): string
 
         return $path;
     }
+
+    private static function prepareStringForNamespace(string $path, string $root = ''): StringHelper
```

Comment:
The PathHelper should follow the same structure as `StringHelper` and `ArrayHelper`. That means making an instance of it with chainable immutable methods.

I'll take care of refactoring this on your branch 👍

Reply:
As discussed in Discord, this is probably the scope of another PR so I close this 😄